stock_reception_screen/models/stock_reception_screen.py

```diff
@@ -350,6 +367,7 @@ def on_barcode_scanned(self, barcode):
 
     def on_barcode_scanned_select_product(self, barcode):
         """Try to find the corresponding product based on the barcode."""
+        self.warn_notification = False
```

Comment:
Instead of resetting `warn_notification` value on each defined step, is it possible to do this in `button_save_step()` method (and also in `button_next_pack()` I suppose), this is the method called each time a user validate a step to go to the next one.